src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -0,0 +1,436 @@
+<template>
+  <div id="dissolution-firm-form">
+    <section class="mt-10">
+      <!-- Dissolution summary -->
+      <v-card flat id="dissolution-summary" class="mt-6">
+        <header class="review-header rounded-t">
+          <v-icon class="ml-2" color="appDkBlue">mdi-domain-remove</v-icon>
+          <label class="font-weight-bold pl-2">Dissolution</label>
+        </header>
+
+        <div class="pb-8" >
+
+          <!-- Association Details -->
+          <section>
+            <AssociationDetails :isSummary="true"/>
+          </section>
+        </div>
+      </v-card>
+    </section>
+
+    <!-- Dissolution Documents Delivery -->
+    <section id="document-delivery-section" class="mt-10">
+      <header>
+        <h2> Documents Delivery</h2>
+        <p class="mt-4 mb-6">
+          Copies of the dissolution documents will be sent to the email addresses listed below.
+        </p>
+      </header>
+      <v-card flat class="mt-6">
+      <DocumentDelivery
+        :class="{ 'invalid-section': isDocumentDeliveryInvalid }"
+        :editableCompletingParty="isRoleStaff"
+        :showCustodianEmail="false"
+        :invalidSection="isDocumentDeliveryInvalid"
+        :registeredOfficeEmail="getBusinessContact.email"
+        :custodianEmail="getDissolutionCustodianEmail"
+        :completingPartyEmail="getUserEmail"
+        :documentOptionalEmail="getDocumentDelivery.documentOptionalEmail"
+        @update:optionalEmail="setDocumentOptionalEmail($event)"
+        @valid="setDocumentOptionalEmailValidity($event)"
+      />
+      </v-card>
+    </section>
+
+    <!-- Folio or Reference Number -->
+    <section id="folio-number-section" class="mt-10" v-if="isPremiumAccount">
+      <header>
+        <h2>Folio or Reference Number for this Filing</h2>
+        <p class="mt-4 mb-6">
+          Enter the folio or reference number you want to use for this filing for your own tracking
+          purposes. The Business Folio or Reference Number is displayed below (if available).
+          Entering a different value below will not change the Business Folio or Reference Number.
+          Only the number below will appear on the transaction report and receipt for this filing.
+        </p>
+      </header>
+       <v-card flat class="mt-6">
+        <TransactionalFolioNumber
+          :accountFolioNumber="getFolioNumber"
+          :transactionalFolioNumber="getTransactionalFolioNumber"
+          :doValidate="getValidateSteps"
+          @change="setTransactionalFolioNumber($event)"
+          @valid="setTransactionalFolioNumberValidity($event)"
+        />
+       </v-card>
+    </section>
+  <!-- Completing Party -->
```

Comment:
Add a blank line between sections. This makes it nice to look at when using code folding feature in IDE. (Also nice to review :) )